server/src/message/data.ts

```diff
@@ -47,20 +47,18 @@ export const MESSAGES: WeatherMessage[] = [
     intensity: 'high',
     message: 'Get a coat before leaving home.',
   },
-
   {
     id: '8',
     type: 'cold',
     intensity: 'normal',
     message: 'Today is not too hot nor too cold, the perfect temperature.',
   },
-
   {
     id: '9',
     type: 'cold',
     intensity: 'low',
     message:
-      'I`m sweating, but the temperature is still ok! Don`t forget to drink water.',
+      "I'm sweating, but the temperature is still ok! Don't forget to drink water.",
```

Comment:
So this was the issue with this string? I couldn't figure it out 😂  Thanks for fixing it :)